pkgs/servers/sql/postgresql/default.nix

```diff
@@ -123,10 +123,16 @@ in {
     sha256 = "09l4zqs74fqnazdsyln9x657mq3wsbgng9wpvq71yh26cv2sq5c6";
   };
 
-  postgresql100 = common {
+  postgresql10 = common {
```

Comment:
I'd keep ```postgresql100```
If you insist on renaming, there are 2 things to consider
 * old name is to be added to ```aliases.nix``` in order not to break user code which uses the old name.
 * why not to rename to ```postgresql_10``` ? ```small-L``` following by ```1``` is not very readable: ![](http://i.imgur.com/HIxqkTw.png)

Reply:
I added a deprecation to aliases.nix, rather than a straight up alias, because I notice that that was the approach taken slightly further down for Ruby when the over-specific names were replaced by ones that actually reflected the versioning policy of Nixpkgs.